content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -1,79 +1,27 @@
 ---
 reviewers:
-- mikedanese
-- luxas
-- errordeveloper
-- jbeda
-title: Using kubeadm to Create a Cluster
+- sig-cluster-lifecycle
+title: Creating a single master cluster with kubeadm
 content_template: templates/task
 ---
 
 {{% capture overview %}}
 
-<img src="https://raw.githubusercontent.com/cncf/artwork/master/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.png" align="right" width="150px">**kubeadm** is a toolkit that helps you bootstrap a best-practice Kubernetes
-cluster in an easy, reasonably secure and extensible way. It also supports
-managing [Bootstrap Tokens](/docs/admin/bootstrap-tokens/) for you and upgrading/downgrading clusters.
+<img src="https://raw.githubusercontent.com/cncf/artwork/master/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.png" align="right" width="150px">**kubeadm** helps you bootstrap a minimum viable Kubernetes cluster that conforms to best practices.  With kubeadm, your cluster should pass [Kubernetes Conformance tests.](https://kubernetes.io/blog/2017/10/software-conformance-certification) Kubeadm also supports other cluster 
```

Comment:
Double whitespaces in `  With kubeadm`?

And change to `should pass [Kubernetes Conformance tests](https://kubernetes.io/blog/2017/10/software-conformance-certification).`?